pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -914,9 +915,38 @@ func handleZbootRestarted(ctxArg interface{}, done bool) {
 	}
 }
 
-func handleInit(networkSendTimeout uint32) {
+
+func handleInit(ctx *zedagentContext, networkSendTimeout uint32) {
 	initializeDirs()
 	handleConfigInit(networkSendTimeout)
+	ctx.configItemSpecMap.AddIntItem("timer.config.interval", "",60, 5, 0xFFFFFFFF, false, true)
```

Comment:
Can you drive this from a set of tables (in global.go; one per type I guess) so that it is easier to review the values compared to the old defaults, and the definitions are close to the defaults.

Also add checks that all items have defaults somehow?

Reply:
Ok. Should be taken care of now with new updates.